tests/testthat/_snaps/list-transpose.md

```diff
@@ -64,3 +64,11 @@
       Error in `list_transpose()`:
       ! `template` must be a character or numeric vector, not a function.
 
+# fail mixing named and unnamed vectors
+
+    Code
+      list_transpose(list(x = list(a = 1, b = 2), y = list(3, 4)))
+    Condition
+      Error in `fn()`:
```

Comment:
Do you think it's worth making this error a bit more clear? At a minimum, it seems like it'd be worth wrapping it to give some context and a more useful `call`.